.travis.yml

```diff
@@ -21,6 +21,9 @@ script:
     else
       echo "Skipping cert renewal for on-demand build"
     fi
+    if [[ "$LOKOLE_QUEUE_BROKER_SCHEME" = "amqp" ]]; then
```

Comment:
@c-w can you explain briefly what this check does?

Reply:
We're currently running the tests in two modes: live and local. The local tests don't reach out to the internet and use exclusively local resources like a RabbitMQ/AMQP backend for Celery running in a container. The live tests do reach out to the internet and also use Azure resources like for example a ServiceBus backend for Celery.

To ensure that the Github authentication is tested during CI, I set `REGISTRATION_CREDENTIALS` as a configuration value in Travis CI to a personal access token for the Ascoder Github service account. To ensure that also the local authentication mode is tested, this check on line 24 disables the Github authentication when we're running in the local test mode so that we fall back to the default username/password authentication mechanism that existed before this pull request.

I've made the distinction between local and live test modes clearer in 27f93a7.